src/dfx/src/main.rs

```diff
@@ -125,7 +125,16 @@ fn main() {
         Err(e) => Err(e),
     };
     if let Err(err) = result {
-        eprintln!("{}", err);
+        for (level, cause) in err.chain().enumerate() {
+            if level == 0 {
+                eprintln!("Error: {}", err);
```

Comment:
It looks like at least one of the e2e tests (failing in CI) will need to be updated to account for this change.

Reply:
https://github.com/dfinity/sdk/runs/4869312230?check_suite_focus=true#step:8:101